batch_seqr_loader/find_inputs.py

```diff
@@ -277,37 +279,75 @@ def splitext_gz(fname: str) -> Tuple[str, str]:
     return base, ext
 
 
-def sm_verify_reads_data(
-    reads_data: List,
-    reads_type: str,
-) -> Union[
-    str, Tuple[List[str], List[str]], None
-]:  # pylint: disable=too-many-return-statements
+@dataclass
+class AlignmentInput:
+    """
+    Sort of a union type for possible alignment inputs
+    """
+
+    bam_or_cram_path: Optional[str] = None
+    index_path: Optional[str] = None
+    fqs1: Optional[List[str]] = None
+    fqs2: Optional[List[str]] = None
+
+
+def sm_verify_reads_data(  # pylint: disable=too-many-return-statements
+    reads_data: Optional[List],
+    reads_type: Optional[str],
+) -> Optional[AlignmentInput]:
     """
     Verify the meta.reads object in a sample db entry
     """
     if not reads_data:
-        logger.error(f'ERROR: no "reads" data')
+        logger.error(f'ERROR: no "meta/reads" field')
+        return None
+    if not reads_type:
+        logger.error(f'ERROR: no "meta/reads_type" field')
+        return None
```

Comment:
Should this be an Exception, it would also be good to include some more information, which sample didn't have reads information.

Reply:
The sample name is printed above in the log, before calling the function. But yeah, worth adding anyway!